glTFLoader/Interface.cs

```diff
@@ -272,7 +283,7 @@ public static Stream OpenImageFile(this Gltf model, int imageIndex, Func<string,
             {
                 var bufferView = model.BufferViews[image.BufferView.Value];
 
-                var bufferBytes = model.LoadBinaryBuffer(bufferView.Buffer, externalReferenceSolver);
+                var bufferBytes = model.LoadBinaryBuffer(bufferView.Buffer, externalReferenceSolver, bufferView.ByteOffset);

```

Comment:
I don't think this is right. The next line (`new MemoryStream...`) line already take the `bufferView.ByteOffset` into account.